Configuration/PyReleaseValidation/python/MatrixReader.py

```diff
@@ -87,7 +89,8 @@ def reset(self, what='all'):
                              'relval_2026':True,
                              'relval_identity':False,
                              'relval_machine':True,
-                             'relval_premix':True
+                             'relval_premix':True,
+                             'relval_nano':False
```

Comment:
@vlimant , I would suggest to enable it for default runTheMatrix . This will allow to run these workflows for IBs. This way we make sure that relvals are working and automatically cache the input used by these wfs.

Reply:
it is partially heavy to run all these, are you sure @smuzaffar that we want to enable them by default ?

Reply:
in https://github.com/cms-sw/cmssw/pull/39472/commits/9806eb497c9a541c7a046a2ba4ea203f1fe6c266